openpype/hosts/fusion/addon.py

```diff
@@ -3,6 +3,15 @@
 
 FUSION_HOST_DIR = os.path.dirname(os.path.abspath(__file__))
 
+FUSION_PROFILE_VERSION = 16
```

Comment:
I'm not entirely sure about hardcoding this number? What happens if next version of Fusion they do remove the 16 or update it to 18 or 19. At that point in time, how do we support both versions of Fusion? We will still need to update the code logic, no? We would need be able to set both.

The logic here looks like it prepares for the variable being different/dynamic but the code doesn't really support that. As such, does it already make sense to add all these lines of code as 'preparation' where it doesn't really prepare is at all?

The code mentions it makes it easy to update the value. But if Fusion 19 comes out and they update it to 19 too then suddenly OpenPype will need to maintain both the 16 variant and the 19 variant to be backwards compatible. So it's not really as easy as just changing this single 16 to 19 since it's break all Fusion 18 projects.

What do you think?